metrics.json

```diff
@@ -1,26 +1,66 @@
 {
```

Comment:
Honestly, it looks too complicated. I would simply put everything in a list like:
```
[
{
"query": "SELECT ROUND(SUM(total_memory_used_size/1024/1024),2) used_memory_mb FROM m_service_memory;",
"name": "hanadb_total_used_memory",
"descritpion": ...
...
}
]
```